tests/tests/swfs/avm2/bitmapdata_draw/test.toml

```diff
@@ -1,2 +1,7 @@
 num_frames = 1
-image = true
+
+[image_comparison]
+tolerance = 0
```

Comment:
Does this actually work? The new `expected.png` is not aliased, unlike the old `expected-linux-Vulkan.png` etc.